docs/api/api_blueprint.apib

```diff
@@ -564,7 +562,49 @@ Get a list of events.
 
 
 ### Create Event [POST]
-Create a new event using a `name`, `starts-at`, `ends-at`, `timzone` and an optional content body.
+Create a new event using a `name`, `starts-at`, `ends-at`, `timezone` and an optional content body.
+
++ name (string) - Name of the event **(required)**
```

Comment:
Why did you remove the table structure and updated description added in https://github.com/fossasia/open-event-orga-server/pull/3832 ?